validator/client/src/main/java/tech/pegasys/teku/validator/client/loader/ValidatorLoader.java

```diff
@@ -96,14 +99,28 @@ public static ValidatorLoader create(
         .collect(toMap(Validator::getPublicKey, Function.identity()));
   }
 
-  private Map<BLSPublicKey, Validator> createExternalSignerValidator(final ValidatorConfig config) {
+  private Map<BLSPublicKey, Validator> createExternalSignerValidator(
+      final ValidatorConfig config, final Supplier<HttpClient> externalSignerHttpClientFactory) {
+    if (config.getValidatorExternalSignerPublicKeys().isEmpty()) {
+      return Collections.emptyMap();
+    }
+
     final Duration timeout = Duration.ofMillis(config.getValidatorExternalSignerTimeout());
+
+    final boolean isReachable =
+        new ExternalSignerUpcheck(
+                externalSignerHttpClientFactory.get(),
+                config.getValidatorExternalSignerUrl(),
+                timeout)
+            .upcheck();
+    STATUS_LOG.externalSignerStatus(config.getValidatorExternalSignerUrl(), isReachable);
```

Comment:
It would probably be good to get this new upcheck in its own PR.  It would also be good to keep retrying - particularly if its failing - otherwise a single error message in the logs can easily be missed and as Paul mentioned, the signer may be up initially but then later go down.  Checking liveness every few minutes probably makes sense.  Would also be good that this is then an async check instead of delaying startup to perform it.

Reply:
moved to separate PR.